### PR TITLE
update digital ocean

### PR DESCRIPTION
**Pull request by maryamsulemani97**
This PR updates `learn/cookbooks/digitalocean_droplet.md`

part of https://github.com/meilisearch/documentation/issues/1719

### CODE REVIEW

File: learn/cookbooks/digitalocean_droplet.md [hunk 1]
```diff
@@ -81,7 +79,7 @@ Configuring your Meilisearch in a **production** environment on DigitalOcean dro
 
 ### 1. Make your domain name point to your droplet
 
-If you want to use your own domain name (or sub-domain), add `A record` in your domain name provider account.
+If you want to use your own domain name (or sub-domain), add a record to your domain name provider account.
```
Comment (maryamsulemani97) [inline learn/cookbooks/digitalocean_droplet.md:82]:
```suggestion
If you want to use your own domain name (or sub-domain), add `A record` in your domain name provider account.
```

File: learn/cookbooks/digitalocean_droplet.md [hunk 2]
```diff
@@ -51,7 +49,7 @@ Tags are a very good method to know who created resources, and for organizing re
 
 ![The search bar, meilisearch, and search-team tags](/digitalocean/06.add-tags.png)
 
-### 7. Finally click on 'Create Droplet'
+### 7. Finally click on "Create Droplet"
```
Comment (guimachiavelli) [inline learn/cookbooks/digitalocean_droplet.md:52]: Not strictly what you're doing here, but that "Finally" bothers me 😅 

```suggestion
### 7. Click on "Create Droplet"
```

**Review by guimachiavelli — APPROVED** [review]
Just one tiny little comment 🐇

**Review by guimachiavelli — APPROVED** [review]
👘

**Pull request merged**